jerry-core/vm/vm.c

```diff
@@ -2009,6 +2009,11 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
 
           goto free_left_value;
         }
+        case VM_OC_SET_FUNCTION_LENGTH:
+        {
+          byte_code_p++;
```

Comment:
Please add a note about that during the execution this instruction is only a NOP.